src/utils/discordEmbed.ts

```diff
@@ -249,7 +249,7 @@ export function getSuccessEmbed(params: {
   title?: string
   description?: string
   thumbnail?: string
-  msg: Message
+  msg?: Message
```

Comment:
make sure this change won't break any other usages

Reply:
this function just decorates `composeEmbedMessage`, it will not break other usages